pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -651,7 +648,7 @@ metadata:
 spec:
   meta: |
     id: createlistrep
-    runNamespace: {{ .Volume.runNamespace }}
+    runNamespace: {{ .Config.JivaRunNamespace.value }}
```

Comment:
if at the end of this runtask, look for successful running of atleast 51% of replicas. If so, consider as success, and patch with only those nodes where replicas are running.

Reply:
Also, reduce the duration and increase the iterations in this task.
Make retry as "6, 20s"

Reply:
Sounds good